context/src/repo/mod.rs

```diff
@@ -53,39 +58,47 @@ impl BundleRepo {
         let mut head_commit_message = None;
         let mut head_commit_author = None;
 
-        // If repo root found, try to get repo details from git.
-        if let Some(git_repo) = bundle_repo_options
-            .repo_root
-            .as_ref()
-            .and_then(|dir| gix::open(dir).ok())
+        #[cfg(feature = "git-access")]
         {
-            bundle_repo_options.repo_url = bundle_repo_options.repo_url.or_else(|| {
-                git_repo
-                    .config_snapshot()
-                    .string_by_key(GIT_REMOTE_ORIGIN_URL_CONFIG)
-                    .map(|s| s.to_string())
-            });
-
-            if let Ok(mut git_head) = git_repo.head() {
-                bundle_repo_options.repo_head_branch = bundle_repo_options
-                    .repo_head_branch
-                    .or_else(|| git_head.referent_name().map(|s| s.as_bstr().to_string()))
-                    .or_else(|| {
-                        Self::git_head_branch_from_remote_branches(&git_repo)
+            // If repo root found, try to get repo details from git.
```

Comment:
This looks like a lot of code change, but it's just indentation (turn white-space diffing off). Using a feature we can prevent including any code for reading git repo details which is important for wasm compilation.